pkg/user/manager/ldap/ldap.go

```diff
@@ -154,7 +180,7 @@ func (m *manager) GetUser(ctx context.Context, uid *userpb.UserId) (*userpb.User
 	}
 	u := &userpb.User{
 		Id:          id,
-		Username:    sr.Entries[0].GetAttributeValue(m.schema.UID),
+		Username:    sr.Entries[0].GetAttributeValue(m.schema.CN),
```

Comment:
here we use the previously fetched cn as the username. they can be configured to the same attribute and the code will work as before